phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/generators/SequentialLoadEventGenerator.java

```diff
@@ -0,0 +1,169 @@
+package org.apache.phoenix.pherf.workload.mt.generators;
+
+import com.beust.jcommander.internal.Maps;
```

Comment:
what's the benefit of not using com.google.common.collect.Maps?

Reply:
Wrong auto import